backend/models/invite_institution.py

```diff
@@ -74,6 +77,41 @@ def send_response(self, current_institution, invitee_key, notification_type, mes
             notification_type=notification_type,
             message=message
         )
+ 
+    def create_notification(self, notification_type, institution_key, receiver_key_urlsafe, user=None):
```

Comment:
change the method send_response_notification to send_reject_response_notification and remove the verification to choose notification_type